selfdrive/car/hyundai/carcontroller.py

```diff
@@ -73,21 +73,32 @@ def update(self, CC, CS):
 
     if self.CP.carFingerprint in HDA2_CAR:
       # steering control
-      can_sends.append(hda2can.create_lkas(self.packer, CC.enabled, self.frame, CC.latActive, apply_steer))
+      if self.CP.carFingerprint in CAR.GENESIS_GV70:
+        can_sends.append(hda2can.create_lfa(self.packer, CC.enabled, self.frame, CC.latActive, apply_steer))
```

Comment:
Can you use lkas? I think the plan is to explore lfa first and move to it in the future (if that's wanted)

Reply:
There was no LKAS traffic for me, which is why I used LFA. A person who was doing a 2022 Tucson port also had this problem. My cluster still shows we're turning on LKAS though, not LFA.